### ISSUE TITLE
Creative Names for Tools

### ISSUE THREAD

**Issue by mottosso** [getavalon/core#375]
### Goal

Simplify conversations about tools and avoid confusing our tools with others, and with verbs.

### Motivation

I'm not 100% convinced by this, but wanted to discuss.

I recently overheard a conversation where a studio had chosen names for their tools that suit their purpose.

- Render Manager
- Rigging Tool
- Displacement Map Generator

The argument was that you don't pick names that run the risk of being confused with the purpose of said tool. That is, when you say "Render Manager", do you mean the specific tool built at this company, or the one in Maya? Or which app are you using, Clarisse?

The proposal then is to pick names that are unique, and not stricly related to their purpose. More like a product or pet name.

- Alysa (for Lys, meaning light in Norwegian/Swedish)
- Pinocchio
- Emboss

As an additional advantage, less techy/dry names makes for better marketing material, with more obvious graphics - such as a picture of Pinocchio in some interesting post for the rigging tool.

Studios tend to do this as well.

- "Tissue", Weta
- "Katana", Imageworks
- "Massive", Weta 
- "Alice", MPC
- "Hyperion", Disney

### Implementation

I don't know! We've got a few tools with generic names.

- Launcher
- Loader
- Creator
- Inventory
- ~Publisher~ Pyblish!

One thing to look out for is that we'd benefit from tools having names that appear *related*. At the moment, because they are all generic, it can be assumed they all follow the same rules (UI, UX, API ec.). One option could be to pick names out of the same comic, like names of individual Transformers or characters from Snow White.

What do you think?

**Comment by tokejepsen** [comment]
Would you not just prefix with Avalon to avoid confusion? So Avalon Launcher and not Ftrack Launcher for example.

**Comment by BigRoy** [comment]
I believe doing it for every tool currently might be a bit too much. The tools that are there now are so specific to Avalon and "make up" Avalon that I'd say the Loader/Creator/Inventory are just that and the "Avalon" name is what Tissue, Katana, etcetera are. It almost feels like making the move to rename Maya's Outliner, Graph Editor and Script Editor to something special.

However, Pyblish - by all means we can refer to it as Pyblish. Since it's also standalone a strong product outside of Avalon.

Does that make a sensible argument?

In short, I do like the idea of stronger naming that stands out yet I doubt the Launcher, Loader, Creator and Inventory will gain much by renaming them too.

**Comment by davidlatwe** [comment]
> It almost feels like making the move to rename Maya's Outliner, Graph Editor and Script Editor to something special.

I have the same feeling about this. But instead of naming them after well known characters or something fancy, how about naming them based on an object or image from life or game ? Would be much low profile as they are actually tools under one working framework, not a single product.

For example:
* Launcher - Bard
* Loader - Market
* Creator - Genie
* Inventory - Pocket
* Project Manager - Merlin (*Merlin* is a well known figure but fit the Avalon)

**Comment by mottosso** [comment]
Thanks for chiming in guys, all good points. Will keep the issue open for a bit in case something else pops up.

**Comment by mkolar** [comment]
> It almost feels like making the move to rename Maya's Outliner, Graph Editor and Script Editor to something special.

I'm seconding this. I don't see them as individual tools, but a subtools of a strong whole. Also for artists it was surprisingly quick to get used to them and I haven't yet seen confusion. Apart from creator. Every artist was asking why the hell is it called creator, when it's not creating anything :). hard one to explain to them I found

**Comment by mottosso** [comment]
> why the hell is it called creator, when it's not creating anything :)

Haha, well why aren't you creating anything for them? Some people use it to generate a pre-defined hierarchy for models and rigs etc. Some use it as a means to create what a Collector can later find, something where they can put things to eventually publish, like an object set with special attributes.

**Comment by mkolar** [comment]
>Haha, well why aren't you creating anything for them?

We are. Well... just object sets mostly which people see more like tagging than creating. But they got used to it. Don't fix it if it ain't broke they say ;)

**Comment by mottosso** [comment]
Feel free to reopen if there's an itch for another stab at this.
